popup.html

```diff
@@ -31,6 +31,7 @@
       color: white;
       border: none;
       cursor: pointer;
+      min-height: 30px;
```

Comment:
min-height to be updated for the quote section not button and min height to be updated to 100